src/tests/index.test.ts

```diff
@@ -136,11 +138,46 @@ it( 'should gracefully handle app ready failures when creating a window on activ
 
 		activate();
 
-		expect( async () => {
-			// Await the mocked `whenReady` promise resolution
+		await new Promise( process.nextTick );
+		expect( createMainWindow ).not.toHaveBeenCalled();
+		expect( captureExceptionMock ).toHaveBeenCalled();
```

Comment:
Seems it's not needed to wrap up this block with `expect`. If the function `whenReady` throws an exception it will make the test to fail.

cc @dcalhoun as you introduced this in https://github.com/Automattic/studio/pull/130.

Reply:
My best guess is that I wrapped it while authoring the test _prior_ to implementing the error catching logic. Removing this makes sense.